be/src/runtime/stream_load/stream_load_record.cpp

```diff
@@ -0,0 +1,137 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "runtime/stream_load/stream_load_record.h"
+
+#include "common/config.h"
+#include "common/status.h"
+#include "rocksdb/db.h"
+#include "rocksdb/slice.h"
+#include "rocksdb/options.h"
+#include "rocksdb/slice_transform.h"
+#include "util/time.h"
+
+
+namespace doris {
+const std::string STREAM_LOAD_POSTFIX = "/stream_load";
+const size_t PREFIX_LENGTH = 4;
+
+StreamLoadRecord::StreamLoadRecord(const std::string& root_path)
+        : _root_path(root_path),
+          _db(nullptr) {
+}
+
+StreamLoadRecord::~StreamLoadRecord() {
+    for (auto handle : _handles) {
+        _db->DestroyColumnFamilyHandle(handle);
+        handle = nullptr;
+    }
+    if (_db != nullptr) {
```

Comment:
This logic is strange. If we need to check `_db` is null here, than the above `_db->DestroyColumnFamilyHandle(handle);` also need to check.

Reply:
> This logic is strange. If we need to check `_db` is null here, than the above `_db->DestroyColumnFamilyHandle(handle);` also need to check.

Done.